test/interop/C/use_PtrConstQualifier.prediff

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+grep printAndReturn lib/PtrConstQualifier.h >> $2
```

Comment:
Printing part of the generated header is smart, I haven't seen that pattern before